RF_Module.pretty/BL653.kicad_mod

```diff
@@ -0,0 +1,108 @@
+  (pad 35 smd rect (at 7.05 1.2 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 36 smd rect (at 6.25 1.235716 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 37 smd rect (at 7.05 2 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 38 smd rect (at 6.25 2.042858 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 39 smd rect (at 7.05 2.8 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 40 smd rect (at 6.25 2.85 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 41 smd rect (at 7.05 3.6 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 42 smd rect (at 7.05 4.4 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 43 smd rect (at 6.25 4.6 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 44 smd rect (at 6.25 3.8 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 45 smd rect (at 5.454546 4.6 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 46 smd rect (at 5.455 3.8 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 47 smd rect (at 4.659092 4.6 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 48 smd rect (at 4.66 3.8 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 49 smd rect (at 3.863638 4.6 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 50 smd rect (at 3.865 3.8 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 51 smd rect (at 3.068184 4.6 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 52 smd rect (at 3.07 3.8 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 53 smd rect (at 2.27273 4.6 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 54 smd rect (at 2.275 3.8 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 55 smd rect (at 1.477276 4.6 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 56 smd rect (at 1.48 3.8 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 57 smd rect (at 0.681822 4.6 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 58 smd rect (at 0.685 3.8 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 59 smd rect (at -0.113632 4.6 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 60 smd rect (at -0.11 3.8 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 61 smd rect (at -0.909086 4.6 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 62 smd rect (at -0.905 3.8 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 63 smd rect (at -1.70454 4.6 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 64 smd rect (at -1.7 3.8 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 65 smd rect (at -2.5 4.6 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 66 smd rect (at -3.85 -4.55 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 67 smd rect (at -5.15 -4.55 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 68 smd rect (at -6.45 -4.55 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
+  (pad 69 smd rect (at -1.75 1.925 180) (size 0.5 0.5) (layers F.Cu F.Paste F.Mask))
```

Comment:
The way I read the drawing, pads 64 and 69 should be flush in x-direction, for example.
Yet there's 50µm offset in the file. Please check.